src/core/schema/contentobject.model.schema

```diff
@@ -45,6 +45,14 @@
       "validators": [],
       "translatable": true
     },
+    "_htmlClassName": {
+      "type": "string",
+      "required": false,
+      "default": "",
+      "inputType": "Text",
+      "validators": [],
+      "help": "If set, this class will be applied to the <html> element"
```

Comment:
```suggestion
      "help": "If set, this class will be applied to the <html> element whenever this is the active page in the course"
```

Reply:
just a thought - anyone know what will happen to < and > when rendered in AAT help tooltip?